tests/virt_autotest/validate_system_health.pm

```diff
@@ -16,9 +16,14 @@ use Utils::Architectures;
 use virt_autotest::common;
 use virt_autotest::utils qw(start_guests check_host_health check_guest_health);
 use virt_utils qw(collect_host_and_guest_logs);
+use alp_workloads::kvm_workload_utils;
+use version_utils qw(is_alp);
```

Comment:
Why is this module adapted to ALP but not scheduled in yaml?

Reply:
added, verification wip

Reply:
Verification passed: http://10.67.129.185/tests/499#
For failure case, feature tests validation has verified that.